src/client/components/common/Card/CardElement/index.tsx

```diff
@@ -1,9 +1,9 @@
 import { FC, ReactNode } from 'react';
-import styled from 'styled-components';
+import styled from 'styled-components/macro';
 
 import { Icon, ArrowDownIcon, IconProps } from '@components/common';
 
-const Container = styled.div<{ width?: string; align?: string; grow?: string; fontWeight?: number }>`
+const Container = styled.article<{ width?: string; align?: string; grow?: string; fontWeight?: number }>`
```

Comment:
The CardElement is a wrapper for all the elements inside card, including every element here:
<img width="674" alt="image" src="https://user-images.githubusercontent.com/78362532/160472353-8fb83a24-0b72-4731-bd20-2e089374c1ee.png">

I think we should leave it as a div

Reply:
an article is a self-contained element - so a card is a good use of article: 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article

Reply:
On a card yes, but in this cases the CardElement is not a card, it is a wrapper not related to the content itself